src/components/PageHeader/PageHeader.tsx

```diff
@@ -19,13 +21,14 @@ const PageHeader = ({
           <Grid col="auto" desktop={{ col: 6 }}>
             {children}
           </Grid>
+
           <Grid col="auto" desktop={{ col: 6 }}>
-            <Search disabled={disableSearch} />
+            {flags && flags.searchComponent ? <Search /> : null}
```

Comment:
Suggestion: I think eventually we do want to remove `disabled` as an option on search. However, removing it at this stage means that the ui will look different than current production with the flag off. Long way to say I think that the Search component should still have it's disabled view and the flag from launch darkly just switches if it's disabled or not. Happy to pair if this comment doesn't make sense. My goal is that when the feature is off the portal looks the same as now, but when the feature is on the search bar is displayed.